### PR TITLE
Fix "vertical" bootstrap 4 form with flexbox enabled

### PR DESCRIPTION
**Pull request by zoidbergwill**
Closes https://github.com/maraujop/django-crispy-forms/issues/573

Hey @carltongibson, can you take a look at this and let me know what you think of this solution?

### CODE REVIEW

File: crispy_forms/tests/test_form_helper.py [hunk 1]
```diff
@@ -711,7 +715,7 @@ def test_bootstrap4_label_class_and_field_class():
 
 
 @only_bootstrap4
-def test_bootstrap4_template_pack():
+def test_template_pack():
     form = TestForm()
     form.helper = FormHelper()
     form.helper.template_pack = 'uni_form'
```
Comment (zoidbergwill) [inline crispy_forms/tests/test_form_helper.py]: `only_bootstrap4` `template_pack = 'uni_form'` O__o

**Comment by carltongibson** [comment]
@zoidbergwill @shulcsm: We happy here?

**Comment by shulcsm** [comment]
Yes, please merge

**Comment by carltongibson** [comment]
Thanks @zoidbergwill!

**Comment by zoidbergwill** [comment]
Thanks @carltongibson, and @shulcsm for helping debug and diagnose the issue. :)

**Pull request merged**